server/configurations/local.yaml

```diff
@@ -125,6 +125,16 @@ s3:
         endpoint:
         region:
         bucket:
+    wasabi-eu-central-2-embeddings:
```

Comment:
Changed this to `wasabi-eu-central-2-derived`. We should probably change the rest of the comments and `embeddings-bucket` config var too.